projects/kit/components/progress/progress-label/progress-label.template.html

```diff
@@ -0,0 +1,2 @@
+<span polymorpheus-outlet class="label" [content]="label"></span>
```

Comment:
Maybe we can just add `<ng-content>` inside it and ditch polymorpheus?

Reply:
done